### PR TITLE
added vmauth to k8s-stack

### PR DESCRIPTION
**Pull request by AndrewChubatiuk**
fixes https://github.com/VictoriaMetrics/helm-charts/issues/829

### CODE REVIEW

File: charts/victoria-metrics-k8s-stack/values.yaml [hunk 1]
```diff
@@ -716,6 +716,31 @@ vmalert:
     #    hosts:
     #      - vmalert.domain.com
 
+vmauth:
+  enabled: false
+  annotations: {}
+  # -- (object) full spec for VMAuth CRD. Allowed values described [here](https://docs.victoriametrics.com/operator/api#vmauthspec)
+  spec:
+    discover_backend_ips: true
```
Comment (f41gh7) [inline charts/victoria-metrics-k8s-stack/values.yaml:724]: vmauth spec doesn't have such setting at spec https://docs.victoriametrics.com/operator/api/

File: charts/victoria-metrics-k8s-stack/values.yaml [hunk 2]
```diff
@@ -716,6 +716,31 @@ vmalert:
...
+    port: "8427"
+  # -- (object) vmagent ingress configuration
+  ingress:
```
Comment (f41gh7) [inline charts/victoria-metrics-k8s-stack/values.yaml:727]: Ingress is part of vmauth.spec

File: charts/victoria-metrics-k8s-stack/values.yaml (hunk 1 above)
Comment (AndrewChubatiuk) [reply charts/victoria-metrics-k8s-stack/values.yaml:724]: it does https://docs.victoriametrics.com/operator/api/#vmauthspec

File: charts/victoria-metrics-k8s-stack/values.yaml (hunk 2 above)
Comment (AndrewChubatiuk) [reply charts/victoria-metrics-k8s-stack/values.yaml]: done

**Review by f41gh7 — APPROVED** [review]
LGTM

**Pull request merged**